dependencies.yaml

```diff
@@ -170,19 +231,29 @@ dependencies:
           - cubinlinker-cu11
           - git+https://github.com/python-streamz/streamz.git@master
           - ptxcompiler-cu11
-          - pyorc
           - rmm-cu11=23.02.*
     specific:
       - output_types: requirements
         matrices:
           - matrix:
               arch: x86_64
             packages:
-              - cupy-cuda115>=9.5.0,<12.0.0a0  # TODO: This might change to cupy-cuda11x?
+              - cupy-cuda115>=9.5.0,<12.0.0a0 # TODO: This might change to cupy-cuda11x?
           - matrix:
               arch: aarch64
             packages:
-              - cupy-cuda11x -f https://pip.cupy.dev/aarch64  # TODO: Verify that this works.
+              - cupy-cuda11x -f https://pip.cupy.dev/aarch64 # TODO: Verify that this works.
+  test_java:
+    common:
+      - output_types: conda
+        packages:
+          # This list is copied from ci/gpu/java.sh and needs to be filtered
+          - dask-cuda=22.12.*
+          - maven
+          - openjdk=8.*
+          - rapids-build-env=22.12.*
```

Comment:
```suggestion
```

`rapids-build-env` can 100% be removed. that package installs several GBs of miscellaneous dependencies.